docs/maintainers/setup_new_repo.md

```diff
@@ -0,0 +1,104 @@
+# Setup Repository
```

Comment:
```suggestion
# Repository Setup
```
applies to the other *Setup x* titles as well.

Reply:
Hmm, this was a bit on purpose because the action always comes first in all the other subtitles (i.e. Create Repository, Setup Repository, etc).

Reply:
In that case `Set up Repository` should be better.